docs/tutorial/part-two/index.md

```diff
@@ -96,238 +50,102 @@ This creates a new site with the following structure.
 │       └── index.js
 ```
 
-This is the minimal setup for a Gatsby site.
-
-To install a plugin, there are two steps. First, you install the plugin's NPM
-package and second, you add the plugin to your site's `gatsby-config.js`.
+#### ✋ Add styles to a css file
 
-Typography.js has a Gatsby plugin, so install that along with some additional required packages by running:
+1. Create a `.css` file in your new project:
 
 ```shell
-npm install --save gatsby-plugin-typography react-typography typography
+cd src
+mkdir styles
+cd styles
+touch global.css
 ```
 
-Next, in your code editor, create a file at the root of your project folder named `gatsby-config.js`.
-This is where you add plugins along with other site configuration.
-
-Copy the following into `gatsby-config.js`
+> Note: You'll need to either stop or restart the running development server, or open a second terminal window. Also, feel free to create these directories and files using your code editor, if you'd prefer.
```

Comment:
If we don't have them start the server, this can be dropped.

Reply:
It's not that -- This has to do with the continuance from part one.

Reply:
Yes, we do. But the dev server from the part one could still be running. we're trying to avoid that. Really this note should be moved to where the new starter is cloned.